src/js/GameMaster.js

```diff
@@ -28,47 +28,56 @@ var GameMaster = (function () {
 			gmVersion = "gamemaster.min";
 		}
 
-		$.getJSON( webRoot+"data/"+gmVersion+".json?v="+siteVersion, function( data ){
```

Comment:
Aside from the change from `$.getJSON` to `$.ajax`, this is all just whitespace changes - see https://github.com/pvpoke/pvpoke/pull/198/files?diff=unified&w=1